src/cli/java/org/commcare/util/screen/QueryScreen.java

```diff
@@ -114,6 +116,8 @@ private String getTitleLocaleStringLegacy() {
             mTitle = Localization.get("case.search.title");
         } catch (NoLocalizedTextException nlte) {
             mTitle = "Case Claim";
+        } catch (NullPointerException npe) {
```

Comment:
Nit: you can make a single `catch` block handle both exceptions. Check out [this code](https://github.com/dimagi/formplayer/blob/4cd1ed39f4c58e2b1b2c5a9f00e38ed3923be2c9/src/main/java/org/commcare/formplayer/util/RequestUtils.java#L68-L73) for an example.

Same comment for the function above.